bin/ch/WScriptJsrt.cpp

```diff
@@ -686,13 +686,21 @@ bool WScriptJsrt::PrintException(LPCSTR fileName, JsErrorCode jsErrorCode)
                 LPCWSTR errorStack = nullptr;
                 size_t errorStackLength = 0;
 
-                IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(_u("stack"), &stackPropertyId), false);
-                IfJsrtErrorFail(ChakraRTInterface::JsGetProperty(exception, stackPropertyId, &stackProperty), false);
+                JsErrorCode errorCode = ChakraRTInterface::JsGetPropertyIdFromName(_u("stack"), &stackPropertyId);
 
-                IfJsrtErrorFail(ChakraRTInterface::JsGetValueType(stackProperty, &propertyType), false);
+                if (errorCode == JsErrorCode::JsNoError)
+                {
+                    errorCode = ChakraRTInterface::JsGetProperty(exception, stackPropertyId, &stackProperty);
+                    if (errorCode == JsErrorCode::JsNoError)
+                    {
+                        errorCode = ChakraRTInterface::JsGetValueType(stackProperty, &propertyType);
+                    }
+                }
 
-                if (propertyType == JsUndefined)
+                if (errorCode != JsErrorCode::JsNoError || propertyType == JsUndefined)
                 {
+                    const char *fName = fileName != nullptr ? fileName : "(Unkown)";
+                    fprintf(stderr, "thrown at %s:\n^\n", fName);
```

Comment:
I would like to keep this one as is since filename is not wchar and cross platform mixed wchar/char output fails. However gonna add comment there to make this point clear!